### DISCUSSION TITLE
Edge dependencies

### DISCUSSION THREAD

**Discussion by ged** [ruby-concurrency/concurrent-ruby#1050] [Q&A] [answered]
`concurrent-ruby-edge` depends on `concurrent-ruby` with a version specification of `~> 1.2.0`, which means that it can't be used with `1.3.1`. If we're using classes from `-edge` should we really stick to `1.2.x` of the other two libraries? Are there plans to update `-edge` so it works with newer versions of the main two libraries? I couldn't find any discussion of it in the changelog or issues, but I might have overlooked something.

**Comment by eregon** [comment, marked as answer]
Thanks for the report, let's discuss this in https://github.com/ruby-concurrency/concurrent-ruby/issues/1052 as it's more visible.
